src/components/NavButton/style.css

```diff
@@ -6,14 +6,18 @@
     background-color: var(--dark-theme-action-button-bg);
     border: none;
     color: var(--dark-theme-action-button-color);
-    padding: 0px;
+    padding: 4px 8px;
 }
 
-.nav-button.action-button:global(.ant-btn:hover),
-.nav-button.action-button:global(.ant-btn:focus) {
+.nav-button.action-button:global(.ant-btn:hover) {
     color: var(--dark-theme-action-button-hover-color);
 }
 
+.nav-button.action-button:global(.ant-btn:focus) {
```

Comment:
Consider changing the `focus` selectors to `focus-visible`, which will make it only respond to keyboard focusing (and not if the user last clicked on the button)

https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible

Reply:
Thank you!

Reply:
FYI: I updated the lines relevant to this PR , but not every place that `:focus` exists in the code base although I have that in mind now for this ongoing work.